examples/fastformers/data/superglue.py

```diff
@@ -0,0 +1,1109 @@
+# coding=utf-8
+# Copyright 2018 The Google AI Language Team Authors and The HuggingFace Inc. team.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
You may want to add Microsoft?

Reply:
Yes, that's right. I will add Microsoft. :) Thanks!